package.json

```diff
@@ -53,7 +53,38 @@
           "description": "Run the linter on save (onSave) or on type (onType)."
         }
       }
-    }
+    },
+    "languages": [
+      {
+        "id": "json",
+        "filenames": [
+          ".pug-lintrc",
+          ".jade-lintrc"
+        ]
+      }
+    ],
+    "jsonValidation": [
+      {
```

Comment:
But it works for me in `vscode-csscomb`, for example, https://github.com/mrmlnc/vscode-csscomb/blob/master/package.json#L76